aad/azure-spring-boot-starter-active-directory/aad-resource-server-by-filter-stateless/README.md

```diff
@@ -26,7 +26,7 @@ to get it working with your Azure AD tenant on both sides.
 
```

Comment:
A stateless or stateful filter is transparent to users. Is there a use case that stateless filter has to play a part?

Reply:
I'm not sure, but when the application is only used to provide an endpoint to return whether the user is in a group, and the token provided comes from a different user each time.
For example, I need to find all users in group1 in `userList`. I may need to traverse the `userList` and use different tokens to access the endpoint. In this case, I might have to use stateless?